underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -131,6 +131,12 @@ public void connectFromWorker(Configuration conf, String hostname) {
   public OutputStream create(String path) throws IOException {
     LOG.debug("Create method: {}", path);
     String newPath = path.substring(Constants.HEADER_SWIFT.length());
+    if (newPath.endsWith("_SUCCESS")) {
```

Comment:
nit: can you please add a brief comment about why we should handle objectname/_SUCCESS separately? is it specific for spark? thanks!